src/pages/Mentor/RoomNav/styles.js

```diff
@@ -0,0 +1,59 @@
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles(
+  () => ({
+    roomText: {
+      marginLeft: "16px",
+      marginRight: "10px",
+      minWidth: "50%",
+      textAlign: "left",
+      paddingTop: "0 !important",
+    },
+
+    listItem: {
+      minHeight: "84px",
+      padding: "16px 32px",
+      cursor: "pointer",
+      display: "flex",
+      justifyContent: "space-between !important",
+    },
+
+    listItemSelected: {
+      minHeight: "84px",
+      padding: "16px 32px",
+      cursor: "pointer",
+      display: "flex",
+      justifyContent: "space-between !important",
+      backgroundColor: "#E9F5E9",
+    },
+    title: {
+      fontSize: "16px !important",
+      color: "#2E2E2E",
+      lineHeight: "24px !important",
```

Comment:
Why we need !important here ??

Reply:
Before adding prefix it was required. I've removed it now

Reply:
@itsomkathe resolve one which you have worked on